src/api/agencyList.ts

```diff
@@ -0,0 +1,11 @@
+export interface Agency {
+  date: string // example - "2019-07-01"
+  operator_ref: number // example - 25,
+  agency_name: string // example - "אלקטרה אפיקים"
```

Comment:
we already have a model for this - https://github.com/hasadna/open-bus-map-search/blob/main/src/model/operator.ts
let's merge them.

Reply:
done